supplementary/alica_ros1/alica_tests/include/test_alica.h

```diff
@@ -48,8 +47,12 @@ struct AlicaTestsEngineGetter
 class AlicaTestFixtureBase : public ::testing::Test
 {
 protected:
+    AlicaTestFixtureBase(){};
+    virtual const char* getHostName() const { return "nase"; }
+    virtual const char* getRoleSetName() const { return "Roleset"; }
+
     std::unique_ptr<alica::AlicaContext> ac;
-    alica::AlicaEngine* ae{nullptr};
```

Comment:
why is this removed?

Reply:
oh, when I was testing I first tried changing this to a unique_ptr, then I saw that it already gets from a unique_ptr and transformed back, then I didn't pay attention when I added this back, thanks.